cmd/tsk/tsk.go

```diff
@@ -93,10 +93,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err := exec.RunTasks(exec.Config, &opts.tasks); err != nil {
-		fmt.Println(err)
-		os.Exit(1)
-	}
+	exec.RunTasks(exec.Config, &opts.tasks)
```

Comment:
this may be called recursively and will this call os.Exit() directly when an command returns an error.